docs/src/PolyhedralGeometry/pg_cones.md

```diff
@@ -12,6 +12,36 @@ Pages = ["pg_cones.md"]
 
 # Cones
 
+
 ## Introduction
 
-## Constructions
+Let $\mathbb{F}$ be an ordered field; the most prominent case here is where $\mathbb{F}=\mathbb{Q}$ are the rational numbers.
+
+A set $C \subseteq \mathbb{F}^n$ is called a (polyhedral) cone if it can be written as the set of nonnegative linear combinations of finitely many vectors in $\mathbb{F}^n$.
+Equivalently, cones can be written as the intersection of finitely many homogeneous linear inequalities.
+
+Any cone is a special case of a polyhedron.
+Conversely, intersecting a cone with a suitable affine hyperplane yields a polyhedron whose faces are in bijection with the faces of the cone.
+Going back and forth between polyhedra and their homogenizations, the cones, is a frequent operation.
+This is one reason for keeping cones as a distinct type.
+
+## Construction
+
+One can construct a cone through an interior description, analogous  to the $V$-description of a polytope. By providing a collection of vectors, the command `positive_hull` will construct the cone generated by those vectors.
+
+```@docs
+positive_hull(::Union{MatElem, AbstractArray{T,2} where T})
```

Comment:
We recently switched to using the type aliases `Vector` and `Matrix` whenever possible instead of `Array`, please adapt to that here, too.
```suggestion
positive_hull(::Union{MatElem, AbstractMatrix{T} where T})
```